app/templates/components/group-nav.hbs

```diff
@@ -9,6 +9,9 @@
       <LinkTo @route="groups.edit.events" @model={{@group.id}} class="item">
         {{t 'Events'}}
       </LinkTo>
+      <LinkTo @route="groups.team" @model={{@group.id}} class="item">
+        {{t 'Team'}}
+      </LinkTo>
```

Comment:
remove team tab, it is giving error, and resolve conflict. UI changes are good 👍🏻